### PR TITLE
:pencil: documented new SDEUtil functionality -mschabel

### PR DESCRIPTION
**Pull request by MSchabel**
do not merge before clearance!

### CODE REVIEW

File: pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md [hunk 1]
```diff
@@ -232,25 +233,62 @@ secretClient
...
 ```javascript
   // import FaaS Toolbelt
-  const { Toolbelt } = require("lp-faas-toolbelt");
+  const { Toolbelt, ConversationContentTypes } = require("lp-faas-toolbelt");
 
   // Create instance
   const conversationUtil = Toolbelt.ConversationUtil();
 
+  // Define Parameters
+  const conversationId = "XXXXXX"
```
Comment (weckbach) [inline pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md]: I don't like XXXXX, can use generate a fake UUID?

File: pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md [hunk 2]
```diff
@@ -377,3 +415,163 @@ conversationUtil.getConversationById(conversationId)
   .then(replacedFiles => //TODO: react on the response)
   .catch(err => //TODO: React to error);
 ```
+### SDE Util
+
+The SDE Util allows to perform SDE related methods, which are listed below. This Util works for **Messaging-Use Cases only**!
```
Comment (weckbach) [inline pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md]: As far as I know, adding SDEs also works for chat. Can you please check that?

File: pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md [hunk 3]
```diff
@@ -377,3 +415,163 @@ conversationUtil.getConversationById(conversationId)
...
+
+```javascript
+  // import FaaS Toolbelt
+  const { Toolbelt } = require("lp-faas-toolbelt");
+
+  // Create instance
+  const sdeUtil = Toolbelt.SDEUtil();
+
+  // Define parameters
+  const visitorId = "XXXXXX";
```
Comment (weckbach) [inline pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md]: Maybe mention that you can get these Ids in the messaging related events

File: pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md (hunk 1 above)
Comment (MSchabel) [reply pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md]: implemented

File: pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md (hunk 3 above)
Comment (MSchabel) [reply pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md]: implemented

**Pull request closed without merging**